### PR TITLE
5600 move docs link to help page

### LINKED ISSUE #5600: Move link to online docs to the `Help` page
Currently, the link to the docs is in our App Drawer. It takes up valuable room - we'll move it to the Help page! The name `Docs` is also a bit technical - User Guide would be better.

EDIT: not a settings accordion section but a new page! see below:

~~Add a new accordion section to the Help page, like this one, called `User Guide`:~~

Inside it, add a section for `Online User Guide (requires internet connection)`, which links to https://docs.muspply.foundation/introduction/introduction

### PR DESCRIPTION
**Pull request by aimee-mcneil-melville**
Fixes #5600 

# 👩🏻‍💻 What does this PR do?

Adds an accordian section to the help page
Within the accordion is a link to the external docs - now named 'User Guide' 
Add label for ```Online User Guide (requires internet connection)```

This links to the external docs

Screenshot 2024-11-29 at 5 32 49 PM

Screenshot 2024-11-29 at 5 32 32 PM

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Go to the help page
- [ ] Click on the User Guide section
- [ ] Click on the link and observe the external docs opening in a new browser tab

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/host/src/Admin/UserGuide.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { ExternalNavLink } from '@common/components';
+import { useTranslation } from '@common/intl';
+import { EnvUtils } from '@common/utils';
+import { ExternalURL, useExternalUrl } from 'packages/config/src';
```
Comment (lache-melvin) [inline client/packages/host/src/Admin/UserGuide.tsx:4]: Just this one will need to be `!@openmsupply-client/config`

File: client/packages/host/src/Help/Help.tsx [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+
+  const [activeSection, setActiveSection] = React.useState<number | null>(null);
+
+  const toggleSection = (index: number) => () =>
+    setActiveSection(activeSection === index ? null : index);
+
   return (
     <Box flex={1} padding={4} sx={{ maxWidth: 800 }}>
       <div>
         <p>Hello</p>
```
Comment (lache-melvin) [inline client/packages/host/src/Help/Help.tsx:24]: can probably remove now 😉

File: client/packages/host/src/Help/Help.tsx [hunk 3]
```diff
@@ -1,20 +1,36 @@
...
       </div>
-
+      <HelpSection
```
Comment (lache-melvin) [inline client/packages/host/src/Help/Help.tsx:26]: Argh apologies for my misunderstanding of the design on Friday - are you okay to update this, no longer an accordion, just the title and then the link? 🙏 

image

**Review by lache-melvin — CHANGES_REQUESTED** [review]
Looks great Aimee! Requesting the change back from accordion (sorry 🙈)

File: client/packages/host/src/Help/Help.tsx (hunk 3 above)
Comment (aimee-mcneil-melville) [reply client/packages/host/src/Help/Help.tsx:26]: Sure no worries!

**Review by lache-melvin — APPROVED** [review]
Sweet thanks Aimee! 🚀

**Pull request merged**